server/api/internal/usecase/interactor/projectAccess.go

```
@@ -100,6 +100,14 @@
 	if err != nil {
 		return "", err
 	}
+	
```

golangci-lint (1.62.2)

File is not `goimports`-ed

(goimports)